### PR TITLE
Fix compilation and tests of pkg/cmd/{experimental,admin}

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: pkg/cmd/admin/groups/new.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func (o *NewGroupOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, arg
 	templateFile := kcmdutil.GetFlagString(cmd, "template")
 	noHeaders := kcmdutil.GetFlagBool(cmd, "no-headers")
 	allowMissingTemplateKeys := kcmdutil.GetFlagBool(cmd, "allow-missing-template-keys")
-	printer, _, err := kubectl.GetPrinter(outputFormat, templateFile, noHeaders, allowMissingTemplateKeys)
+	printer, _, err := printers.GetStandardPrinter(outputFormat, templateFile, noHeaders, allowMissingTemplateKeys, nil, nil, nil)
```
Comment (sttts) [inline pkg/cmd/admin/groups/new.go]: @ncdc do you know what to pass here? The copier is the scheme. Which RESTMapper makes sense and which codecs?

Comment (ncdc) [reply pkg/cmd/admin/groups/new.go]: @sttts can you just use `f.PrinterForCommand(cmd)`?

Comment (sttts) [reply pkg/cmd/admin/groups/new.go]: At least it's type correct.

File: pkg/cmd/experimental/config/patch.go [hunk 2]
```diff
@@ -137,7 +137,7 @@ func (o *PatchOptions) RunPatch() error {
 	rawExtension := &runtime.Unknown{
 		Raw: originalPatchedObjJS,
 	}
-	printer, _, err := kubectl.GetPrinter("yaml", "", false, true)
+	printer, _, err := printers.GetStandardPrinter("yaml", "", false, true, nil, nil, nil)
```
Comment (sttts) [inline pkg/cmd/experimental/config/patch.go]: @ncdc here is another one.

File: pkg/cmd/admin/groups/new.go [hunk 3]
```diff
@@ -5,12 +5,12 @@ import (
...
 
+	"github.com/spf13/cobra"
+
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/pkg/kubectl"
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
-
-	"github.com/spf13/cobra"
+	"k8s.io/kubernetes/pkg/printers"
```
Comment (ncdc) [inline pkg/cmd/admin/groups/new.go]: I've been calling this kprinters

File: pkg/cmd/experimental/config/patch.go (hunk 2 above)
Comment (ncdc) [reply pkg/cmd/experimental/config/patch.go]: See pkg/cmd/cli/cmd/export.go

File: pkg/cmd/admin/groups/new.go (hunk 3 above)
Comment (sttts) [reply pkg/cmd/admin/groups/new.go]: fixed

File: pkg/cmd/experimental/config/patch.go (hunk 2 above)
Comment (sttts) [reply pkg/cmd/experimental/config/patch.go]: fixed

**Pull request merged**